### DISCUSSION TITLE
Upgrade Kong OSS to Paid Plan

### DISCUSSION THREAD

**Discussion by yaser-aibees** [Kong/kong#13683] [Help]
I am using Kong OSS on my debian server which sits between Cloudflare and Kubernetes cluster. Is there a way to upgrade the OSS version to paid plan? I want to use Rate Limiting Advanced plugin which is only available on paid plan and I don't want to migrate to Konnect platform.

**Comment by chronolaw** [comment]
This page may be helpful: https://konghq.com/pricing

**Comment by yaser-aibees** [comment]
That's not gonna work for OSS version, I assume.
